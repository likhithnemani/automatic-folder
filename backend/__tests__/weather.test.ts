Content for backend/__tests__/weather.test.ts
