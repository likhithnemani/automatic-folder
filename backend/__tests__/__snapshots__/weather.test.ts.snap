Content for backend/__tests__/__snapshots__/weather.test.ts.snap
